src/main/java/com/rabbitmq/client/impl/ClientVersion.java

```diff
@@ -28,10 +29,18 @@
 
     static {
         version = new Properties();
+        InputStream inputStream = ClientVersion.class.getClassLoader()
+                .getResourceAsStream("version.properties");
         try {
-            version.load(ClientVersion.class.getClassLoader()
-              .getResourceAsStream("version.properties"));
+            version.load(inputStream);
         } catch (IOException e) {
+        } finally {
+            try {
+                if(inputStream!=null) {
```

Comment:
Fixed. Sorry, I had to disable my formatter as it formatted things differently :) And thanks for the quick replies!